docs/UserGuide.md

```diff
@@ -41,146 +37,167 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `addTask completeUserGuide d/28-Feb-2017 t/2400 p/high #Important`
 
-### 2.3. Listing all persons : `list`
+### Adding a floating task: `addFTask`
+
+Adds a floating task to the task manager<br>
+Format: `addFTask FLOATINGTASKNAME [p/PRIORITY] [#TAG]...`
+
+> Floating Tasks can have any number of tags (including 0)
+> Floating Tasks can only have one priority
+> Floating Tasks cannot have any deadlines 
+
+Examples:
 
-Shows a list of all persons in the address book.<br>
-Format: `list`
+* `addFTask reachDiamondRank p/low #Overwatch`
 
-### 2.4. Editing a person : `edit`
+### Adding an event: `addEvent`
 
-Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Adds an event to the task manager<br>
+Format: `addEvent EVENTNAME sd/START DATE ed/END DATE [st/START TIME] [et/END TIME] [p/PRIORITY]  [#TAG]...`
 
-> * Edits the person at the specified `INDEX`.
-    The index refers to the index number shown in the last person listing.<br>
+> Events can have any number of tags (including 0)
+> Events can only have one priority
+> Events must have a start and end date
+> If no start time or end time specified, system will assume start at 0000 on start date and end at 2359 on end date
+
+Examples:
+
+* `addEvent finishCS3230Lab sd/01-Mar-17 ed/01-Mar-17 st/2000 et/2200 p/med #CS3230`
+
+### Editing a task : `edit`
+
+Edits an existing task in the task manager.<br>
+Format: `edit INDEX [t/DEADLINE TIME] [p/PRIORITY] [#TAG]...`
+
+> * Edits the task at the specified `INDEX`.
+    The index refers to the index number shown in the last task listing.<br>
     The index **must be a positive integer** 1, 2, 3, ...
 > * At least one of the optional fields must be provided.
 > * Existing values will be updated to the input values.
-> * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-> * You can remove all the person's tags by typing `t/` without specifying any tags after it. 
+> * When editing tags, the existing tags of the task will be removed i.e adding of tags is not cumulative.
+> * You can remove all the task's tags by typing `#` without specifying any tags after it. 
 
 Examples:
 
-* `edit 1 p/91234567 e/johndoe@yahoo.com`<br>
-  Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@yahoo.com` respectively.
+* `edit 1 t/9pm p/high #CS1010`<br>
+  Edits the deadline, priority and hashtag of the 1st task to be `9pm` , `high` and `#CS1010` respectively.
 
-* `edit 2 Betsy Crower t/`<br>
-  Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `edit 2 t/9pm p/high `<br>
+  Edits the name of the 2nd task's deadline to `9pm` and priority to `high` and clears all existing tags.
 
```

Comment:
As your parameters are optional, more thinking can be placed for this `edit` function.
Maybe you can explain more, like what will happen to the existing values that are not specified in a particular `edit` command.
Eg. edit 2 Assignment
- What will happen to the dates that were earlier on but not included now?
- Or what if I intend to leave out a start time to convert it from a task that only have deadline, etc.